_python_workout_code/bowl.py

```diff
@@ -0,0 +1,15 @@
+class Bowl():
+    MAX =  3
+    def __init__(self):
+        self.bowl = []
+    
+    def add_scoops(self, *args):
+        for i in args:
+            if len(self.bowl) < self.MAX:
+                try:
+                    self.bowl.append(i.flavor)
+                except:
+                    pass
```

Comment:
you could also add print here to see, which data was processed while failing the operation and the exception type.